python/cugraph-dgl/examples/dataset_from_disk_cudf.ipynb

```diff
@@ -23,8 +23,9 @@
     "import rmm\n",
     "import torch\n",
     "from rmm.allocators.torch import rmm_torch_allocator\n",
-    "# rmm.reinitialize(pool_allocator = True, initial_pool_size = 15e9, maximum_pool_size=24e9)\n",
-    "# torch.cuda.memory.change_current_allocator(rmm_torch_allocator)"
+    "rmm.reinitialize(initial_pool_size=15e9)\n",
+    "#rmm.mr.set_current_device_resource(rmm.mr.CudaAsyncMemoryResource(initial_pool_size=15e9))\n",
```

Comment:
did you intend to leave in this commented-out code?

Reply:
Yeah, This is to allow users to tweak to a different allocator if need arises like running into OOM due to fragmentation. Will add a comment explaining it.